### PR TITLE
Refactor/code issues

### PR DESCRIPTION
**Pull request by vinitkumar**
@Engineering-Robert Does it make much sense to hardcode this in form? 

https://github.com/Engineering-Robert/public-wifi-speeds/blob/960a333a1614ebe8218ac3eb3854634e11e994b0/src/components/CitySelection.js#L78

### CODE REVIEW

File: src/components/AddEntryForm.js [hunk 1]
```diff
@@ -30,21 +31,12 @@ class AddEntryForm extends Component {
...
-            dd = "0" + dd
-        }
-        if (mm < 10) {
-            mm = "0" + mm
-        }
-        let date = dd + "/" + mm + "/" + yyyy
-        return date
+    getCurrentDate = () => {
+        // "02/01/2018"
+        return moment(new Date()).format('DD/MM/YYYY')
```
Comment (robertcoopercode) [inline src/components/AddEntryForm.js:37]: Oh, this is nice! I've never used Moment before.

File: src/components/App.js [hunk 2]
```diff
@@ -8,10 +8,10 @@ import { Route } from "react-router-dom"
 import PropTypes from "prop-types"
 
 const App = (props, { authUser }) => (
-    <React.Fragment>
+    <Fragment>
```
Comment (robertcoopercode) [inline src/components/App.js:11]: Oh, wasn't aware you could do this. We will have to change for all instances of <React.Fragment>

File: package.json [hunk 3]
```diff
@@ -15,7 +15,8 @@
     "react-places-autocomplete": "^5.4.3",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.0.14",
-    "react-social-login-buttons": "^1.3.5"
+    "react-social-login-buttons": "^1.3.5",
+    "speedtest-net": "^1.3.2"
```
Comment (robertcoopercode) [inline package.json]: Did you want to add the implementation of the speedtest library to the app as well?

**Comment by robertcoopercode** [comment]
@vinitkumar In regards the CitySelection component, i'm open to any suggested improvements to the onClick handler function. I suppose we could loop through each dropdown item passing in the name of the city and corresponding coordinates.
https://github.com/Engineering-Robert/public-wifi-speeds/blob/960a333a1614ebe8218ac3eb3854634e11e994b0/src/components/CitySelection.js#L78

File: package.json (hunk 3 above)
Comment (vinitkumar) [reply package.json]: Nope, it is not required here.

File: src/components/App.js (hunk 2 above)
Comment (vinitkumar) [reply src/components/App.js:11]: I can do this in this same PR

File: src/components/AddEntryForm.js (hunk 1 above)
Comment (vinitkumar) [reply src/components/AddEntryForm.js:37]: Moment is super handy. Once you have used it, you never have to deal with dates on your own again.

File: package.json [hunk 4]
```diff
@@ -14,7 +15,7 @@
     "react-places-autocomplete": "^5.4.3",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.0.14",
-    "react-social-login-buttons": "^1.3.5"
+    "react-social-login-buttons": "^1.3.5",
```
Comment (robertcoopercode) [inline package.json]: @vinitkumar A trailing comma like this in a JSON object causes an error.

**Comment by robertcoopercode** [comment]
Thanks @vinitkumar!

**Pull request merged**